### DISCUSSION TITLE
404 File not found

### DISCUSSION THREAD

**Discussion by Trompillon** [community/community#72936] [New to GitHub] [answered]
Hi!

I'm new to github and a beginner to code.
I upload what i expected to be my first internet site but i got this message : 

**404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.**

I know that is a current error, i searched and find people having the same issue, i tried my best to fix all the issues by myself but i don't get it! 

Here's my repository : https://github.com/Trompillon/Chez-Trompi
And the link : https://trompillon.github.io/Chez-Trompi/

(I tried to add index.html at the end but it do not work)

Thank you by advance! and sorry for being a noob!

**Comment by apartyagi** [comment]
wait a minut i'll solve your problem

**Comment by apartyagi** [comment, marked as answer]
Hello, buddy!

Please check out this link. You'll see your site working, but you'll also notice a styling error because your CSS isn't loading. You need to change or correctly link your CSS.

[website](https://trompillon.github.io/Chez-Trompi/R%C3%A9cit%20de%20campagne/Accueil/)

"Please mark this as the answer."

Is there anything else I can assist you with?

**Reply by Trompillon** [reply]
Thank You very much!

I'm gonna check what's wrong with my css but that's another problem

Thank you again!

**Reply by Trompillon** [reply]
The problem is fixed! style.css was written with a Maj (Style.css) now it is working! I need just now to add the correct links in my code and it will be finished, thank you so much again!

**Comment by BioModelLab** [comment]
Hi, I need help finding my account in Google how, and I cannot add existing GitHub as a collaborator to some of my projects. please help.

**Reply by Lapidis2** [reply]
cany also help me to deploy my front end project am getting 404 error below is my github
https://github.com/Lapidis2/My_Brand_Frontend

**Reply by apartyagi** [reply]
Hello Tell me what is issue are you facing in detail i will help you
